package.json

```diff
@@ -27,8 +27,8 @@
     ]
   },
   "dependencies": {
-    "@rsksmart/rif-relay-common": "1.0.0-alpha.1",
-    "@rsksmart/rif-relay-contracts": "1.0.0-alpha.0",
+    "@rsksmart/rif-relay-common": "github:rsksmart/rif-relay-common#PP-504/remove-worker-configuration",
+    "@rsksmart/rif-relay-contracts": "github:rsksmart/rif-relay-contracts#PP-504/remove-worker-configuration",
```

Comment:
I'm guessing that those two libs should be merged before this one and these two lines changed back increasing the tag, right?

Reply:
oops spamming sorry